RFC-0001-prsestissimo-metrics.md

```diff
@@ -0,0 +1,251 @@
+# Prestissimo Runtime Metrics Capturing and reporting
+
+## Proposers
+
+* Ahana, an IBM company.
+* Uber.
+  
+## Prototype PR
+There are two approaches in this PR:https://github.com/prestodb/presto/pull/21599 which will be discussed in detail.
+
+## Summary
+
+Propose design to capture Prestissimo runtime metrics and expose these metrics to a popular time series DB, Prometheus.
+Also, discuss a general way to capture and serialize metrics to any custom format.
+
+## Background
+### Velox runtime metrics:
```

Comment:
The library exposes metrics of the HTTP server it creates ( like last pull), cpu overhead of pull etc. If we reuse the presto servers endpoint, we might need to add/measure those